src/vmm/src/vmm_config/mmds.rs

```diff
@@ -37,3 +38,26 @@ impl Display for MmdsConfigError {
         }
     }
 }
+
+/// Keeps the IMDS configuration.
+#[derive(Deserialize)]
+#[serde(deny_unknown_fields)]
+pub struct ImdsConfig {
+    /// IMDS configured version
+    pub imds_version: ImdsVersion,
+}
+
+impl ImdsConfig {
+    /// Returns the IMDS version.
+    pub fn imds_version(&self) -> ImdsVersion {
+        self.imds_version
+    }
+}
+
+impl From<ImdsVersion> for ImdsConfig {
+    fn from(version: ImdsVersion) -> Self {
+        ImdsConfig {
+            imds_version: version,
+        }
+    }
+}
```

Comment:
We don't really have an `IMDS`, we only have a `imds_version` field on `MMDS` which describes the `MMDS` capabilities.

I don't think we should have an `ImdsConfig` struct; just add `imds_version: ImdsVersion` field to `struct MmdsConfig`.

Reply:
We sync-ed offline and decided to move config specific to MMDS to `mmds` crate, beacuse it's not possible to have `imds_version` as part of `MmdsConfig`.